src/js/oncoprint.js

```diff
@@ -0,0 +1,125 @@
+var Track = require('./track').Track;
+var _ = require('underscore');
+var d3 = require('d3');
+var $ = require('jquery');
+module.exports = {};
+
+// TODO: use self everywhere
+
+var defaultOncoprintConfig = {
+	cell_width: 10,
+	cell_padding: 3,
+	render: 'table',
+};
+
+function Oncoprint(container_selector_string, config) {
+	var self = this;
```

Comment:
I'd like to understand the advantage of `this.approach` (haha) over

```js
function Oncoprint(con,fig,uration) {
    var self = {};
    self.foo = "bar";
    return self;
}
```

This article goes in depth on JavaScript's `this` keyword and while there are times to use this (jQuery does cool things by using it to get the context of an event automatically), I don't think this is one of those times. http://javascriptissexy.com/understand-javascripts-this-with-clarity-and-master-it/